backend-go/api/codepair/v1/models/model_run_follow_up_request.go

```
@@ -0,0 +1,13 @@
+package models
+
+type RunFollowUpRequest struct {
+
+	// Document ID to apply the follow-up
+	DocumentId string `json:"documentId"`
```

golangci-lint (1.62.2)

[warning] var-naming: struct field DocumentId should be DocumentID

(revive)